en/docs/guides/user-mgt/ask-password.md

```diff
@@ -1 +1,100 @@
-# Users Self Sign Up
+# Creating Users Using the Ask Password and Email Verification Options
+
+This section is about the user on-boarding flows initiated by
+administrators which allows respective end users to decide their own
+passwords or verify the accounts created by administrators. 
+
+## On-boarding with Ask password
+
+This process is initiated by the administrator when selecting **Ask
+password from user** during the user creation process. This is different
+from the default flow, in which the administrator decides the passwords
+for users. Using the **Ask Password** option is the standard method for
+user management as the administrator does not have to specify or remember passwords when creating an account for a user.
+
```

Comment:
you can add horizontal lines in between headings to make the page look nicer. 

------